builtin/common/misc_helpers.lua

```diff
@@ -517,6 +517,66 @@ function table.shuffle(t, from, to, random)
 end
 
 
+function table.find(t, e, i, j)
+    i = i or 1
```

Comment:
broken indentation - please use tabs, this is as easy as "Convert to tabs" in most editors

Reply:
Done.